src/container/sidebar.jsx

```diff
@@ -333,17 +335,17 @@ export const options = [
     },
     {
         title: "Billett administrasjon",
-        icon: faTicketAlt,
+        icon: faTicket,
```

Comment:
why?

Reply:
![image](https://github.com/phoenixlan/phoenixcrew-v1/assets/30247084/20fdd8ab-ff8e-463b-9aa0-ead63edb6e17)
Because `faTicket` looks better than `faTicketAlt`. Just look at the image! Ticket voucher has faTicketAlt, and it looks bad without the white border inside the icon.

Reply:
Ok!

Worth noting, normal ticket will look ugly at low resolutions.

Reply:
But fair enoguh